src/legacy/core_plugins/kibana/public/context/app.html

```diff
@@ -25,31 +32,34 @@
   <div class="kuiInfoPanel kuiInfoPanel--error kuiVerticalRhythm">
     <div class="kuiInfoPanelHeader">
       <span class="kuiInfoPanelHeader__icon kuiIcon kuiIcon--error fa-warning"></span>
-      <span class="kuiInfoPanelHeader__title">
-        Failed to load the anchor document
-      </span>
+      <span
+        class="kuiInfoPanelHeader__title"
+        i18n-id="kbn.context.failedToLoadAnchorDocumentDescription"
+        i18n-default-message="Failed to load the anchor document"
+      ></span>
     </div>
 
     <div class="kuiInfoPanelBody">
       <div
         class="kuiInfoPanelBody__message"
         ng-if="contextApp.state.loadingStatus.anchor.reason === contextApp.constants.FAILURE_REASONS.INVALID_TIEBREAKER"
+        i18n-id="kbn.context.noSearchableTiebreakerFieldDescription"
+        i18n-default-message="No searchable tiebreaker field could be found in the index pattern {indexPatternId}. Please change the advanced setting {tieBreakerFields} to include a valid field for this index pattern."
+        i18n-values="{
+          indexPatternId: contextApp.state.queryParameters.indexPatternId,
+          tieBreakerFields: '<code>context:tieBreakerFields</code>'
```

Comment:
Due to issue #26173, currently to treat a value as HTML, `html_` prefix is needed.
```suggestion
          html_tieBreakerFields: '<code>context:tieBreakerFields</code>'
```